src/ipc/ipc4/handler.c

```diff
@@ -999,6 +1002,8 @@ void ipc_cmd(struct ipc_cmd_hdr *_hdr)
 	if (err)
 		tr_err(&ipc_tr, "ipc4: %d failed err %d", target, err);
 
+	ipc_platform_ack_cmd(ipc_get());
```

Comment:
Is it possible that IPC request is forwarded to another core in non-blocking way (like set_pipeline_state) while this ack would be performed by the primary core before the IPC downlink mailbox is consumed?